packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -287,20 +309,32 @@ export async function sideloadAddIn(manifestPath: string, app?: OfficeApp, canPr
   if (isDesktop && app) {
     if (isSideloadingSupportedForDesktopHost(app)) {
       await registerAddIn(manifestPath);
-      sideloadFile = await generateSideloadFile(app, manifest, document);
+      if (app == OfficeApp.Outlook) {
+        sideloadFile = await getOutlookExePath();
+      } else {
+        sideloadFile = await generateSideloadFile(app, manifest, document);
+      }
     } else {
       throw new Error(`Sideload is not supported for ${app} on ${AppType.Desktop}.`);
     }
   } else {
```

Comment:
I think it would be better to make this else be `else if (isWeb && app)`. You then could remove other checks for whether this is web from inside the clause.

Reply:
Are you referring to this part being "else is"?  I am not clear from your comment:

 else {
    if (platform && platform == AppType.Web && document === undefined && app && isSideloadingSupportedForWebHost(app)) {
      throw new Error(`For sideload to web, you need to specify a document url.`);
    }

    if (app && !isSideloadingSupportedForWebHost(app)) {
      throw new Error(`Sideload to web is not supported for ${app}.`);
    }
    
    const manifestFileName: string = path.basename(manifestPath);
    sideloadFile = await generateSideloadUrl(manifestFileName, manifest, document, isTest);
  }

Reply:
Sorry if not clear, there is `if (isDesktop && app) { ... }` and then after that can be `else if (isWeb && app)`.  I think at this point `app is required` so it might make sense to do a check `if (!app)` and throw an error about it. The point is that I think if you look over this it can be simplified.

Reply:
I think the `platform && platform == AppType.Web` can be simplified to `const isWeb == ...`. Of course, it also might be better to error out if there is no platform specified, and then use a switch statement for `switch (platform)` `case  AppType.Desktop:` and AppType.Web. Lots of ways this code could be improved.

Reply:
Made some changes here - let me know if you aren't OK with them